common/scripts/babel-conditional-preprocess.js

```diff
@@ -81,6 +81,10 @@ exports.default = babelHelper.declare((_api, opts) => {
         Handle(path, featureSet, stabilizedFeatureSet);
       },
 
+      TSMethodSignature(path) {
```

Comment:
@dmceachernmsft Can we do this as a separate PR?
With explanation to why we need to do this?
And possibly an example in the conditional compilation examples doc?

Reply:
@prprabhu-ms has a seperate PR for this as well I just included it so mine will build. its in the same spot so it wont conflict. (hopefully). https://github.com/Azure/communication-ui-library/pull/1913

Reply:
Merged mine. You can merge from `main` and get rid of this diff now.

Thanks for helping make conditional compilation exhaustive... slowly... incrementally.... some day. ;-)